main.cpp

```diff
@@ -75,29 +75,44 @@ struct <#structName1#>                                //4
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float Doughnuts { 4.2f }, Days { 3.2f };
```

Comment:
The naming convention for all variables and functions is to start them with a lower-case letter. 
Only type names begin with a capital letter.
- [ ] revise